src/main/application-constants.ts

```diff
@@ -1,10 +1,11 @@
 import appConfig from './application-config'
-import { dirname, join } from 'path'
+import path, { dirname, join } from 'path'
```

Comment:
you can remove the `path` here and use only `join` instead of `path.join` bellow.
```suggestion
import { dirname, join } from 'path'
```

Reply:
You're right, thank you